doc/AddASetting.md

```diff
@@ -1,40 +1,40 @@
 # Adding a Settings Property
 
-1. Add to wincon.w
-    * THIS IS NOT IN OPENCONSOLE. Make sure you update
-      `.../console/published/wincon.w` in the OS repo when you submit the PR.
+### 1. Add to wincon.w
+* THIS IS NOT IN OPENCONSOLE. Make sure you update
+      `.../console/published/wincon.w` in the Open Source repository when you submit the Pull Request.

```

Comment:
Actually, this is supposed to be "OS". I'm referring to the actual _Windows_ OS repo here, this isn't a change that can be made entirely in the open-source repo here.

Reply:
Oh, my mistake. I'll remember that. Changing it now.